src/main.rs

```diff
@@ -125,6 +140,49 @@ fn main() {
         startup_nsh
             .write_all(include_bytes!("startup.nsh"))
             .unwrap();
+
+        // Create user provided additional files
+        for files in additional_files {
```

Comment:
```suggestion
        for file in additional_files {
```
Use singular form in iteration